recipe/meta.yaml

```diff
@@ -66,7 +66,7 @@ outputs:
         - libcurand-dev      # [(cuda_compiler_version or "").startswith("12")]
       run:
         - python
-        - {{ pin_compatible('numpy') }}
+        - numpy
```

Comment:
Picked up this change from PR: https://github.com/conda-forge/implicit-feedstock/pull/83

Should relax the version constraint on `numpy`. Given it is not explicitly specified in `host`, likely the latest `numpy` is installed there (and then the constraint set here is too stringent). Since `numpy` is not linked against either, `pin_compatible` need not be used here. So this drops it, which should allow more flexibility when install `implicit` and `numpy` into user environments

Reply:
Thanks @jakirkham ! I think this will fix the issue in https://github.com/benfred/implicit/issues/679 -